### ISSUE TITLE
acorn-es7-plugin requires Acorn v2, 3 or 4

### ISSUE THREAD

**Issue by djMax** [MatAtBread/nodent#82]
Not sure exactly what's going on yet, but all of the sudden I'm getting this error:

```
Error: acorn-es7-plugin requires Acorn v2, 3 or 4 (While processing preset: "/mydev/node_modules/babel-preset-gasbuddy/index.js")
    at module.exports (/mydev/node_modules/acorn-es7-plugin/acorn-es7-plugin.js:11:15)
    at Object.<anonymous> (/mydev/node_modules/nodent/lib/parser.js:173:28)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/mydev/node_modules/nodent/nodent.js:13:14)
```

`npm ls acorn`

```mydev@1.0.0 /mydev
├─┬ babel-preset-gasbuddy@2.1.1
│ └─┬ fast-async@6.2.1
│   └─┬ nodent@3.0.14
│     └── acorn@5.0.0
└─┬ eslint-config-gasbuddy@1.0.2
  └─┬ eslint@3.18.0
    └─┬ espree@3.4.0
      ├── acorn@4.0.4
      └─┬ acorn-jsx@3.0.1
        └── acorn@3.3.0```

**Comment by djMax** [comment]
This makes sense - Acorn current is v5, nodent specifies that it's fine with 5 (>2.x), gets 5, and then acorn-es7-plugin throws because it doesn't want 5. Anecdotally, telling acorn-es7-plugin to allow 5 seems to "work" though not sure how to test exhaustively.

**Comment by matAtWork** [comment]
Thanks for pointing that out - I'll run tests, etc., and confirm operation with Acorn 5 before fixing the error.

**Comment by matAtWork** [comment]
Fixed by acorn-es7-plugin v1.1.6 (https://github.com/MatAtBread/acorn-es7-plugin/commit/ac8e89bc1fcf5436fe675b6c611ce8ee1a1c4ae9) and nodent v3.0.15 (https://github.com/MatAtBread/nodent/commit/2defe0f88e1e1aff7990d37f503e47a53166f422)
